tests/functional/ui/test_error_improvements.py

```
@@ -161,3 +218,238 @@ def deprecated_storage_class_creation_rules(self, setup_ui_class):
+        request.addfinalizer(finalizer)
+        return created_pools
+
+    @pytest.fixture()
+    def ec_io_cleanup(self, request):
+        """
+        Tracks StorageClass and Secret objects created for IO verification
+        and cleans them up in finalizer.
+        """
+        resources = []
+
+        def finalizer():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)